_posts/kubernetes/2018-07-19-Adding-an-Artifact-Account.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
@TCCurrie S3 support has been added to Halyard since the question was posted to the custom config workaround is no longer needed for OSS users. However, we can probably re-work the answer so that it's Armory specific. Something like
```
### Answer
If you're using Armory Spinnaker, add an S3 Artifact account to `clouddriver-local.yml` using [this config](https://docs.armory.io/install-guide/adding_accounts/#s3).

If you're using OSS Spinnaker, you can add an S3 account [these Halyard commands](https://www.spinnaker.io/reference/halyard/commands/#hal-config-artifact-s3). 
```

Reply:
Here are some links for `More Resources`
* https://docs.armory.io/install-guide/adding_accounts/#adding-artifact-accounts
* https://www.spinnaker.io/reference/artifacts/

Reply:
great! Done  Will commit all of them at once.